app/assets/stylesheets/components/common/_social-sharing.scss

```diff
@@ -0,0 +1,19 @@
+.social-sharing {
+  .social-sharing__title {
```

Comment:
We've found that because the class names are already pretty specific, we don't actually need to nest them like this. The output is larger (although it probably gzip's out) and would just keep it inline with the rest of the CSS. Could you move the styles out so they're not nested?